src/shogun/mathematics/linalg/LinalgBackendEigen.h

```diff
@@ -127,6 +128,18 @@ class LinalgBackendEigen : public LinalgBackendBase
 		return c;
 	}
 
+	/** Eigen3 matrix C = alpha*A + beta*B method */
+	template <typename T>
+	SGMatrix<T> add_impl(const SGMatrix<T>& a, const SGMatrix<T>& b, T alpha, T beta) const
```

Comment:
true but think about performance as well... what about having the option for both?